example/BugTrackerExample/Bug.cs

```diff
@@ -87,7 +87,7 @@ void SendEmailToAssignee(string message)
 
         public string ToDotGraph()
         {
-            return DotGraphFormatter.Format(_machine.GetInfo());
+            return new DotGraphFormatter().ToDotGraph(_machine.GetInfo());
```

Comment:
This was one of the earlier refactorings we went for, I think - on the public API, there's no need to allocate a `DotGraphFormatter` object for the operation, since there's nothing that can be done with the instance except the one method call. Internally, allocating seems fine for this if it simplifies the implementation.